cpp/doxygen/main_page.md

```diff
@@ -47,7 +47,7 @@ mamba install -c rapidsai-nightly -c conda-forge libkvikio
 # Create new environment (CUDA 11.8)
 mamba create -n libkvikio-env -c rapidsai-nightly -c conda-forge python=3.12 cuda-version=11.8 libkvikio
 # Create new environment (CUDA 12.5)
-mamba create -n libkvikio-env -c rapidsai-nightly -c conda-forge python=3.12 cuda-version=12.5 libkvikio
+mamba create -n libkvikio-env -c rapidsai-nightly -c conda-forge python=3.12 cuda-version=12.8 libkvikio
```

Comment:
The comment above this line should also be changed (currently reads "CUDA 12.5"). And similar changes a few lines up.

To avoid mistakes like that in the future, I think it'd probably be better to just change those parentheticals to `(CUDA 11)` and `(CUDA 12)`.

This needs to be updated too, now that the file it references is being removed from the repo:

https://github.com/rapidsai/kvikio/blob/716a99f212ce2e0f9862bad1be81a12c56d03dc6/docs/source/install.rst#L48-L49

Reply:
Improved in e83f99c.